test/TestUtils.ts

```diff
@@ -46,7 +46,7 @@ export async function startRelay (
   args.push('--relayHubAddress', relayHubAddress)
   const configFile = path.resolve(__dirname, './server-config.json')
   args.push('--config', configFile)
-  args.push('--stakeManagerAddress', stakeManager.address)
+  // args.push('--stakeManagerAddress', stakeManager.address)
```

Comment:
If this is not needed, remove. If needed, do not touch. This is not relevant to release to the slightest.